src/proto/graplinc/grapl/api/event_source/v1beta1/event_source.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package graplinc.grapl.api.event_source.v1beta1;
+
+import "graplinc/common/v1beta1/types.proto";
+
+message CreateEventSourceRequest {
```

Comment:
Gonna need `message` level comments to satsify the linters

Reply:
The lints appear to pass: https://buildkite.com/grapl/grapl-verify/builds/3733#0181a619-8e63-481c-b178-f3adc1ea62ca